core/storage/exploration/gae_models.py

```diff
@@ -239,6 +239,53 @@ def has_reference_to_user_id(cls, unused_user_id):
         return False
 
 
+class MathExplorationImagesModel(base_models.BaseModel):
+    """Temporary Storage model for storing information useful while generating
+    images for math rich-text components in explorations.
+
+    The id of each instance is the id of the corresponding exploration.
+    """
+
+    # A boolean which indicates if there are images saved in datastore for all
+    # the math rich-text components in an exploration.
+    math_images_exist = ndb.BooleanProperty(indexed=True, default=False)
```

Comment:
Use a better name to indicate whether we need to generate svg images for the corresponding exploration.

Reply:
done